app/controllers/mixins/middleware_common_mixin.rb

```diff
@@ -9,10 +8,19 @@ def index
     redirect_to :action => 'show_list'
   end
 
+  def show
+    return unless init_show
+    show_middleware
+  end
+
   private
 
-  def display_name
-    ui_lookup(:tables => @record.class.base_class.name)
+  def display_name(display = nil)
+    if display.blank?
```

Comment:
@mzazrivec : what is the plan with UI lookup?

Will the remaining `ui_lookup` calls be handled in background by gettext? or will we replace the `ui_lookup` calls in code?
Or do you plan to keep the `.yaml` file?

I am asking because here we will have new code using `ui_lookup` and it might make sense to get rid of it as we move forward rather then create new calls ...